### PR TITLE
Insights Docs Fixes.

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
## Jira Ticket
 https://jira.d2iq.com/browse/D2IQ-82456

### CODE REVIEW

**Comment by kaiwalyajoshi** [comment]
PR Preview Page: http://docs-d2iq-com-pr-4251.s3-website-us-west-2.amazonaws.com/dkp/kommander/2.2/insights/

File: pages/dkp/kommander/2.2/insights/index.md [hunk 1]
```diff
@@ -11,24 +11,34 @@ techPreview: true
...
 
-DKP Insights works seamlessly within networked or air-gapped environments. It collects metrics, logs, and events from multiple data sources and uses rule-based heuristics to detect issues from the data and generate insight items for the users.
+Access the DKP Insights Dashboard by selecting **Insights** from the sidebar menu.
 
-You can filter the insight items for a selected cluster or project by:
+DKP Insights consists of two components:
+- DKP Insights Management: Runs on the Kommander [Management](../clusters/) cluster.
+- DKP Insights Engine: Runs on each [Attached](../clusters/) Kubernetes clusters.
 
+DKP Insights Engine needs be to explictly enabled on each Attached cluster, instructions are found in the section on [Insights Setup and Configuration](./insights-setup/).
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:26]:
```suggestion
DKP Insights Engine needs be to explicitely enabled on each Attached cluster, instructions are found in the section on [Insights Setup and Configuration](./insights-setup/).
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
-     branch: main
-    timeout: 20s
-    url: https://github.com/mesosphere/insights-catalog-applications
-   EOF
-   ```
-
-   This provides all attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.
-
-1. Enable the DKP Insights engine on each attached or managed cluster from the DKP UI by selecting: Cluster -> Applications -> Options Menu -> Enable -> Deploy with defaults.
+DKP Insights Management is installed by default on your Mangement Cluster.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:12]:
```suggestion
DKP Insights Management is installed by default on your Management Cluster. 
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 3]
```diff
@@ -11,24 +11,34 @@ techPreview: true
 
 <p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
 
-We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as checking for security issues, monitoring resource usage, verifying that workloads and clusters follow best practices, and resolving common issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
+DKP Insights is intended to assist Kubernetes Administrators with routine tasks:
+- Resolving common issues.
+- Monitoring resource useage.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:16]:
```suggestion
- Monitoring resource usage.
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 4]
```diff
@@ -11,24 +11,34 @@ techPreview: true
 
 <p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
 
-We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as checking for security issues, monitoring resource usage, verifying that workloads and clusters follow best practices, and resolving common issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
+DKP Insights is intended to assist Kubernetes Administrators with routine tasks:
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:14]:
```suggestion
DKP Insights assists Kubernetes Administrators with routine tasks:
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 5]
```diff
@@ -11,24 +11,34 @@ techPreview: true
...
+- Checking security issues.
+- Verifying workloads and clusters follow best practices.
 
-DKP Insights works seamlessly within networked or air-gapped environments. It collects metrics, logs, and events from multiple data sources and uses rule-based heuristics to detect issues from the data and generate insight items for the users.
+Access the DKP Insights Dashboard by selecting **Insights** from the sidebar menu.
 
-You can filter the insight items for a selected cluster or project by:
+DKP Insights consists of two components:
+- DKP Insights Management: Runs on the Kommander [Management](../clusters/) cluster.
+- DKP Insights Engine: Runs on each [Attached](../clusters/) Kubernetes clusters.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:24]:
```suggestion
- DKP Insights Engine: Runs on each [Attached](../clusters/) Kubernetes cluster.
```

File: pages/dkp/kommander/2.2/insights/index.md (hunk 1 above)
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:26]:
```suggestion
DKP Insights Engine must be enabled explicitely on each Attached cluster. You can find instructions in the [Insights Setup and Configuration](./insights-setup/) section.
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 6]
```diff
@@ -11,24 +11,34 @@ techPreview: true
...
 
+DKP Insights Engine needs be to explictly enabled on each Attached cluster, instructions are found in the section on [Insights Setup and Configuration](./insights-setup/).
+
+DKP Insights Engine collects events and metrics on the Attached cluster, and uses rules-based heuristics on potential problems of varying criticality so they can be quickly identified and resolved. These Insights are then forwarded and displayed in the DKP Insights Dashboard.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/index.md:28]:
```suggestion
DKP Insights Engine collects events and metrics on the Attached cluster, and uses rule-based heuristics on potential problems of varying criticality, so they can be quickly identified and resolved. These Insights are then forwarded and displayed in the DKP Insights Dashboard.
```

File: pages/dkp/kommander/2.2/insights/insights-viewing/index.md [hunk 7]
```diff
@@ -1,16 +1,17 @@
...
 enterprise: false
 techPreview: true
 ---
 <p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis.</p>
 
-Select one of your workspaces to get started viewing your insight items. From a Workspace Dashboard, the Insights summary table, below the workspace summary cards, displays the most recent insight items. You can toggle the view between Critical, Warning, Notices, and View All for a summary of insight items.
+
+Once the [Insights Engine is Enabled](../insights-setup/) select one of your workspaces to view your insight items. From a Workspace Dashboard, the Insights summary table, below the workspace summary cards, displays the most recent insight items. You can toggle the view between Critical, Warning, Notices, and View All for a summary of insight items.
 
 Select Insights from the sidebar menu for the DKP Insights dashboard, where you can filter the insight items several different ways:
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-viewing/index.md:16]:
```suggestion
Select Insights from the sidebar menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 8]
```diff
@@ -1,43 +1,46 @@
...
-    url: https://github.com/mesosphere/insights-catalog-applications
-   EOF
-   ```
-
-   This provides all attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.
-
-1. Enable the DKP Insights engine on each attached or managed cluster from the DKP UI by selecting: Cluster -> Applications -> Options Menu -> Enable -> Deploy with defaults.
+DKP Insights Management is installed by default on your Mangement Cluster. 
+
+DKP Insights Engine needs to be enabled manually on each Attached cluster to receive insight items on the Insight Dashboard.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:14]:
```suggestion
Enable the DKP Insights Engine manually on each Attached cluster to receive insight items on the Insights Dashboard.
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 9]
```diff
@@ -1,43 +1,46 @@
...
+
+## Enable the DKP Insights Engine
+
+
+1. Add DKP Insights Engine Addon
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:19]:
```suggestion
1. Add the DKP Insights Engine Addon 
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 10]
```diff
@@ -1,43 +1,46 @@
...
+DKP Insights Management is installed by default on your Mangement Cluster. 
+
+DKP Insights Engine needs to be enabled manually on each Attached cluster to receive insight items on the Insight Dashboard.
+
+## Enable the DKP Insights Engine
+
+
+1. Add DKP Insights Engine Addon 
+
+   Apply yaml to your Management cluster from the CLI
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:21]:
```suggestion
   Apply yaml to your Management cluster from the CLI:
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 11]
```diff
@@ -1,43 +1,46 @@
...
+    kommander.d2iq.io/workspace-default-catalog-repository: "true"
+spec:
+  interval: 1m0s
+  ref:
+    branch: main
+  timeout: 20s
+  url: https://github.com/mesosphere/insights-catalog-applications
+EOF
+``` 
+   This grants all Attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:41]:
```suggestion
   This grants all Attached clusters the ability to enable the Insights Engine. The DKP Insights icon appears under Applications.
```

**Review by gvasquezvargas — COMMENTED** [review]
Addressed some error warnings from the linter, so it should let us merge once the suggestions are commited. Left some other suggestions in terms of language.

**Comment by msdolbey** [comment]
@kaiwalyajoshi - I made some of these spelling fixes - I also saw that one of these pages wasn't appearing, and I think something got a little weird in one of the headers (like there were two lines that said `beta: false` or something like that).

Either way, you'll want to pull before committing any new changes.

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 12]
```diff
@@ -1,43 +1,46 @@
...
+
+2. Enable the DKP Insights Engine on each Attached cluster from the DKP UI by selecting: Cluster -> Applications -> Options Menu -> Enable -> Deploy with defaults.
```
[the remark this answers is not in the document]
Comment (Colin1841) [reply pages/dkp/kommander/2.2/insights/insights-setup/index.md]: Yea, that's better. good catch, I was going to get back to this and forgot about it!

**Review by Colin1841 — APPROVED** [review]

**Review by gvasquezvargas — APPROVED** [review]

File: pages/dkp/kommander/2.2/insights/index.md [hunk 13]
```diff
@@ -9,26 +9,40 @@ enterprise: false
...
-<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
+<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only.</p>
 
-We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as checking for security issues, monitoring resource usage, verifying that workloads and clusters follow best practices, and resolving common issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
+DKP Insights assists Kubernetes Administrators with routine tasks:
 
-DKP Insights works seamlessly within networked or air-gapped environments. It collects metrics, logs, and events from multiple data sources and uses rule-based heuristics to detect issues from the data and generate insight items for the users.
+- Resolving common issues.
+- Monitoring resource usage.
+- Checking security issues.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/insights/index.md:18]:
```suggestion
- Checking security issues
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 14]
```diff
@@ -9,26 +9,40 @@ enterprise: false
...
+- Verifying workloads and clusters follow best practices.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/insights/index.md:19]:
```suggestion
- Verifying workloads and clusters follow best practices
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 15]
```diff
@@ -9,26 +9,40 @@ enterprise: false
...
 
-You can filter the insight items for a selected cluster or project by:
+Access the DKP Insights Dashboard by selecting **Insights** from the sidebar menu.
+
+DKP Insights consists of two components:
+
+- DKP Insights Management: Runs on the Kommander [Management](../clusters/management-cluster) cluster.
+- DKP Insights Engine: Runs on each [Attached](../clusters/) Kubernetes cluster.
+
+DKP Insights Engine needs be to explicitly enabled on each Attached cluster, instructions are found in the section on [Insights Setup and Configuration](./insights-setup/).
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/insights/index.md:28]:
```suggestion
You must enable the DKP Insights Engine explicitly on each Attached cluster. You can find instructions in the section on [Insights Setup and Configuration](./insights-setup/).
```

File: pages/dkp/kommander/2.2/insights/index.md [hunk 16]
```diff
@@ -9,26 +9,40 @@ enterprise: false
...
+
+DKP Insights Engine collects events and metrics on the Attached cluster, and uses rule-based heuristics on potential problems of varying criticality, so they can be quickly identified and resolved. These Insights are then forwarded and displayed in the DKP Insights Dashboard.
+
+Within the DKP Insights Dashboard, insights can be filtered for a selected cluster or project by:
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/insights/index.md:32]:
```suggestion
Within the DKP Insights Dashboard, you can filter insights for a selected cluster or project by:
```

File: pages/dkp/kommander/2.2/insights/insights-setup/index.md [hunk 17]
```diff
@@ -1,43 +1,45 @@
...
 
-To enable DKP Insights:
+## Enable the DKP Insights Engine
 
-1.  Apply yaml on your management cluster from the CLI:
+1.  Add DKP Insights Engine Addon
 
-   ```bash
-   export WORKSPACE_NAMESPACE=kommander
+    Apply yaml to your Management cluster from the CLI
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/insights/insights-setup/index.md:20]: Is applying this YAML file the way to accomplish what's named in the numbered step? if so, consider this wording:

Add the DKP Insights Engine Addons by applying the following YAML from the CLI:

    ```yaml
     <code>
    ```

**Review by JohnB64-JBV — CHANGES_REQUESTED** [review]

**Comment by Colin1841** [comment]
Merging into the other PR, we can pick up edits from there now.

**Pull request merged**